### PR TITLE
DYN-5835: Deliver the .net6 dynamocoreruntime builds

### PR DESCRIPTION
**Pull request by sm6srw**
### Purpose

This pull request does:
* add a new template for a net6 windows core package
* remove windows binaries from the existing service net6 runtime

I new job, `Dynamo>DynamoSelfServ>net60_windows_target-release` has been added for daily releases on `master-5`

### Declarations

Check these if you believe they are true

- [X] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated
- [ ] This PR contains no files larger than 50 MB 

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: tools/NuGet/template-net60-core/DynamoVisualProgramming.ServiceCoreRuntime.nuspec [hunk 1]
```diff
@@ -17,8 +17,6 @@
         </dependencies>
     </metadata>
     <files>
-        <!--we expect that the base path is bin\NET60_Linux\Release\-->
-        <file src="..\..\NET60_Windows\Release\**" target="lib\net6.0\windows\" />
         <file src="**" target="lib\net6.0\linux\" />
```
Comment (pinzart90) [inline tools/NuGet/template-net60-core/DynamoVisualProgramming.ServiceCoreRuntime.nuspec:20]: we probably need to change some of the build steps too to get rid of the extra work done for delivering the windows part right ?

Comment (pinzart90) [reply tools/NuGet/template-net60-core/DynamoVisualProgramming.ServiceCoreRuntime.nuspec:20]: also the target folder could be simplified if we only ever deliver for a single target framework/os.
Not critical though

**Review by pinzart90 — APPROVED** [review]

**Comment by mjkkirschner** [comment]
actually @sm6srw I'm a bit confused.
Does this nuspec and job deliver the Dynamo.Net6ALL sln or just the core sln?

**Comment by sm6srw** [comment]
Ths build script we use for the new job builds Net6all.lsn.

**Pull request merged**